### PR TITLE
[MOS-259] Fix no calllog entries in DND mode

### PR DESCRIPTION
**Pull request by SP2FET**
**Description**

Fixed no calllog entries in DND mode, which were caused
by not starting the call in DB interface, so no entry was created.

Moreover, small optimizing of the DND handling.

**Your checklist for this pull request**

Make sure that this PR:
- [x] Complies with our guidelines for contributions
- [ ] Has unit tests if possible.
- [x] Has documentation updated

### CODE REVIEW

**Comment by SP2FET** [comment]
This will probably have to wait with merging until we'll fix MOS-602, as it will create multiple calllog entries while rejecting..
.

File: module-services/service-cellular/call/CallMachine.hpp [hunk 1]
```diff
@@ -143,10 +148,16 @@ namespace call
     {
         void operator()(Dependencies &di, const call::event::CLIP &clip, CallData &call)
         {
+            // the order here is crucial - rejecting by modem is first, because of how DND works -
+            // we have to reject incomming call ASAP.
```
Comment (Lefucjusz) [inline module-services/service-cellular/call/CallMachine.hpp:152]: "incoming" ;)

**Review by Lefucjusz — APPROVED** [review]

File: module-services/service-cellular/call/CallMachine.hpp (hunk 1 above)
Comment (SP2FET) [reply module-services/service-cellular/call/CallMachine.hpp]: thanks, fixed

**Comment by SP2FET** [comment]
closing due to duplicate

**Pull request closed without merging**